ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4524,6 +4524,16 @@ PDSCV:
   - CONTROL
 
 
+PDDCV:
```

Comment:
Should be able to use `PDSCV` as discharge and supply damper is the same thing on a VAV.

Reply:
For VAVs specifically, they only have one damper so it can be thought of as controlling the air discharged into the space in addition to controlling the air supplied from the AHU into the VAV, which is why `PDSCV` can be used. If this were an AHU directly feeding a single zone then you would be correct to use discharge. Hope that helps.